BaseTools/Source/Python/Trim/Trim.py

```diff
@@ -281,11 +300,14 @@ def DoInclude(Source, Indent='', IncludePathList=[], LocalSearchPath=None, Inclu
                         F = File.readlines()
                 break
         else:
-            EdkLogger.error("Trim", "Failed to find include file %s" % Source)
-            return []
-    except:
-        EdkLogger.error("Trim", FILE_OPEN_FAILURE, ExtraData=Source)
-        return []
+    # MU_CHANGE [BEGIN]
+            EdkLogger.error("Trim",FILE_NOT_FOUND, ExtraData="Failed to find include file %s" % Source)

```

Comment:
Nit, but this is a good time to fix it.

```suggestion
	            EdkLogger.error("Trim", FILE_NOT_FOUND, ExtraData="Failed to find include file %s" % Source)
```

Reply:
Updated!